__tests__/posts.ts

```diff
@@ -7971,3 +7972,488 @@ describe('query post awards', () => {
     ]);
   });
 });
+
+describe('mutation startPostBoost', () => {
+  const MUTATION = `
+    mutation StartPostBoost($postId: ID!, $duration: Int!, $budget: Int!) {
+      startPostBoost(postId: $postId, duration: $duration, budget: $budget) {
+        _
+      }
+    }
+  `;
+
+  const params = { postId: 'p1', duration: 7, budget: 5000 };
+
+  const runTest = async <T = unknown>(execute: () => Promise<T>) => {
```

Comment:
For some reason, setting value in our redis io pool during `beforeEach` does not work.

Reply:
Ended up not needing this anymore since we don't need to check the balance 😭

Reply:
🙈